app/views/main.scala.html

```diff
@@ -12,42 +12,77 @@
         <title>@title</title>
         <link href='//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700' rel='stylesheet' type='text/css'>
         <link rel="stylesheet" href="@routes.Assets.at("lib/normalize.css/normalize.css")">
-        <link rel="stylesheet" href="@routes.Assets.at("lib/foundation/css/foundation.min.css")">
         <link rel="stylesheet" href="@routes.Assets.at("css/main.min.css")">
         <script src="@routes.Assets.at("lib/modernizr/modernizr.min.js")"></script>
         <link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/favicon.png")">
     </head>
     <body>
-        <header class="fw-wrapper">
-            <div class="row">
-                <div class="small-12 columns">
-                    <a href="#" class="logo">
-                        @svg.conductrFullColor()
-                    </a>
+        <div class="off-canvas-wrap" data-offcanvas>
+            <div class="inner-wrap">
+                <!-- Off Canvas Menu -->
+                <div class="left-off-canvas-menu">
                     <nav>
-                        <a href="#">Menu Item</a>
-                        <a href="#">Menu Item</a>
+                        <a href="/conductr">Menu Item</a>
+                        <a href="/conductr">Menu Item</a>
                     </nav>
+
+                    <!--TO DO, Christoper can we pull in the TOC Aside here also-->
+                    <aside>
+                      <h3><a href="/conductr">Toc Item One</a></h3>
+                      <h3><a href="/conductr" class="active">Toc Item Two</a></h3>
+                      <ul>
+                        <li><a href="/conductr/install.html">Toc Sub Item One</a></li>
+                        <li><a href="/conductr/quickstart.html">Toc Sub Item Two</a></li>
+                        <li><a href="/conductr/quickstart.html">Toc Sub Item Two</a></li>
+                      </ul>
+                      <h3><a href="/conductr">Toc Item Three</a></h3>
+                      <h3><a href="/conductr">Toc Item Four</a></h3>
+                    </aside>
+                    <!--TO DO, Christoper can we pull in the TOC Aside here also-->
```

Comment:
Shouldn't this be an index concern and not a main concern? Again this page is cross project...